ChangeLog.md

```diff
@@ -6,6 +6,7 @@ This project's release branch is `master`. This log is written from the perspect
 
 * nixpkgs-overlays
   * Remove override of acme module that pinned it to the version in nixpkgs-20.03. This is used for automatic https certificate provisioning.
+* Fix a bug in `Obelisk.Route.Frontend` where `routeLink`, `routeLinkDynAttr`, and `dynRouteLink` would not behave exactly like `<a href="...">` when run by JavaScript. These functions now open a new tab when Ctrl-clicked.
```

Comment:
The wording here makes it seem like _all_ discrepancies between plain link behavior and route link behavior have been resolved. Let's be a little bit more modest, because there might be weird corner cases we have missed. All we have done is make ctrl click work as expected. 0:-)

Reply:
Actually, these are not my words, there was already a [Changelog message](* Fix a bug in `Obelisk.Route.Frontend` where `routeLink`, `routeLinkDynAttr`, and `dynRouteLink` would not behave exactly like `<a href="...">` when run by JavaScript. These functions now scroll to the top of the page when the link is clicked. ([#540](https://github.com/obsidiansystems/obelisk/pull/540))) like this before.

Anyhow, I'll change it.

Reply:
Done.